openassessment/xblock/ui_mixins/mfe/assessment_serializers.py

```diff
@@ -85,12 +123,21 @@ class SubmittedResponseSerializer(Serializer):
                 fileSize: (Bytes?)
                 fileIndex: (Integer, positive)
             }
-        ]
+        ],
+        teamUploadedFiles: Null
     }
     """
 
     textResponses = SerializerMethodField()
     uploadedFiles = SerializerMethodField()
+    teamUploadedFiles = NullField(source="*")
+
+    def __init__(self, instance=None, *args, **kwargs):  # pylint: disable=keyword-arg-before-vararg
+        # Very weird workaround to control serialization for None input as data
```

Comment:
I've seen middleware workarounds for this. Did we (collectively) write one / is it worth the effort?